tvalmetrics/__init__.py

```diff
@@ -1 +1 @@
-from tvalmetrics.rag_scores_calculator import RagScoresCalculator  # noqa: F401
+from tvalmetrics.rag_scores_calculator import TonicValidateCalculator  # noqa: F401
```

Comment:
I think we should keep this named `RagScoresCalculator` to avoid breaking changes in this PR.
```suggestion
from tvalmetrics.rag_scores_calculator import RagScoresCalculator  # noqa: F401
```